web/themes/joinup/joinup_theme.theme

```diff
@@ -39,24 +42,35 @@ function joinup_theme_preprocess_page(&$variables) {
   // Set page title.
   $title = \Drupal::service('title_resolver')->getTitle($request, $route_match->getRouteObject());
 
+  // Check if page is form page and page content has no white background.
+  $is_form_container = FALSE;
+
   // We need to show the header in a blue variant on certain routes:
   // node and rdf entity forms, user forms.
   $route_name = $route_match->getRouteName();
   $blue_header_routes = [
+    'adms_validator.adms_validator_form',
     'collection.propose_form',
+    'contact_form.contact_page',
     'entity.node.edit_form',
     'entity.rdf_entity.edit_form',
-    'node.add',
-    'node.add_page',
     'rdf_entity.propose_form',
-    'rdf_entity.rdf_add',
```

Comment:
I see your point, as I mentioned I have an idea some forms are out of the workflow. Still, the button is there and the forms are reachable, so please keep them in the list for now.